docs/source.rst

```diff
@@ -92,50 +92,47 @@ We recommend conducting all research related to your submission in Tor Browser.
 If you are unsure whether you are using Tor, you can visit the address
 https://check.torproject.org.
 
-All organizations operating SecureDrop have a *Landing Page* that provides their
-own organization-specific recommendations for using SecureDrop. We encourage you
-to consider an organization's *Landing Page* before submitting to them.
+All organizations operating SecureDrop have a landing page that provides their own organization-specific recommendations for using SecureDrop. We encourage you to consider an organization’s landing age before submitting to them.
 
 .. note::
 
    Each SecureDrop instance is operated and administered independently by
    the organization you are submitting to. Only the journalists associated
    with that organization can see your submissions.
 
-Most organizations make their *Landing Page* prominently accessible from their
+Most organizations make their SecureDrop prominently accessible from their
 main website's homepage (for news organizations, typically under sections called
 "Tips" or "Contact us"). You can also find an incomplete list of organizations
 accepting submissions through SecureDrop in the `SecureDrop Directory`_
 maintained by Freedom of the Press Foundation.
 
-Using Tor Browser, find the ".onion" address for the *Source Interface* of
+Using Tor Browser, find the ".onion" address for the SecureDrop for
 the organization that you wish to submit to.
 
 .. tip::
 
    If the organization does have an entry in the SecureDrop Directory, we
    recommend comparing the address of the entry with the one on the
-   organization's own *Landing Page*.
+   organization's own SecureDrop landing page.
 
    If the two addresses don't match, please do not submit to this organization
    yet. Instead, please `contact us <https://securedrop.org/report-an-error>`__
-   through the SecureDrop Website, using Tor Browser. For additional
+   through the SecureDrop website, using Tor Browser. For additional
    security, you can use our .onion service address in Tor:
 
    ``secrdrop5wyphb5x.onion/report-an-error``
```

Comment:
while we're in here:
``sdolvtfhatvsysc6l34d65ymdwxcujausv7k5jk4cy5ttzhjoi6fzvyd.onion/report-an-error``